pkg/apis/servicecatalog/validation/serviceclass.go

```diff
@@ -94,29 +105,32 @@ func validateClusterServiceClassSpec(spec *sc.ClusterServiceClassSpec, fldPath *
 		allErrs = append(allErrs, field.Required(fldPath.Child("clusterServiceBrokerName"), "clusterServiceBrokerName is required"))
 	}
 
+	commonErrs := validateCommonServiceClassSpec(&spec.CommonServiceClassSpec, fldPath, create)
+
+	if len(commonErrs) != 0 {
+		allErrs = append(commonErrs)
+	}
+
+	return allErrs
+}
+
+func validateCommonServiceClassSpec(spec *sc.CommonServiceClassSpec, fldPath *field.Path, create bool) field.ErrorList {
+	commonErrs := field.ErrorList{}
+
 	if "" == spec.ExternalID {
-		allErrs = append(allErrs, field.Required(fldPath.Child("externalID"), "externalID is required"))
+		commonErrs = append(commonErrs, field.Required(fldPath.Child("externalID"), "externalID is required"))
 	}
 
 	if "" == spec.Description {
-		allErrs = append(allErrs, field.Required(fldPath.Child("description"), "description is required"))
+		commonErrs = append(commonErrs, field.Required(fldPath.Child("description"), "description is required"))
```

Comment:
Just for the record, I don't think this name change is necessary (renaming `allErrs`). No need to do this one in the future.